src/redux/modules/transactionsList/selectors.js

```diff
@@ -0,0 +1,23 @@
+
+export const transactionsTotalsSelector = state =>
+  state.transactionsList.transactions;
+
+
+
+export const sumOfallInputsSelector = state =>
+  transactionsTotalsSelector(state)
+    .map(trns => trns.inputs)
+    .flat()
+    .map(input => input.output_value)
+    .reduce((inVal, inAcc) => inVal + inAcc, 0) / 1000;
+
+export const sumOfallOutputsSelector = state =>
+  transactionsTotalsSelector(state)
+    .map(trns => trns.outputs)
+    .flat()
+    .map(output => output.value)
+    .reduce((outVal, outAcc) => outVal + outAcc, 0) / 1000;
+
+
```

Comment:
calculation `sumOfallInputsSelector ` and `sumOfallOutputsSelector ` seems the same, do you have any ideas show you can helper function and then reuse it in both scenarios?

Reply:
why name is `sumOfallInputsSelector` not `sumOfAllInputsSelector `?

Reply:
I have removed these selectors due to I will not use them. Instead I will use a helper function and useSelector hook;